vent/api/actions.py

```diff
@@ -1,4 +1,5 @@
 import ast
+import docker
```

Comment:
I don't think this is required, since it's getting the client instance from `Plugin` which does the import, right?